test/functional-portable/corerp/noncloud/resources/recipe_terraform_test.go

```diff
@@ -172,6 +174,10 @@ func Test_TerraformRecipe_KubernetesPostgres(t *testing.T) {
 							{ID: "/planes/kubernetes/local/namespaces/corerp-resources-terraform-pg-app/providers/core/Service/postgres"},
 						},
 					},
+					/*{
+						Name: secretName,
+						Type: validation.SecretStoresResource,
+					},*/
```

Comment:
Are these to be deleted?

Reply:
will enable these to test functional test updates once bicep PR is merged in.